README.md

```diff
@@ -0,0 +1,69 @@
+# Next Video [![Circle CI](https://circleci.com/gh/Financial-Times/n-video.svg?style=svg)](https://circleci.com/gh/Financial-Times/n-video)
+
+Creates a video player and attaches analytics
+
+## Installation
+
+    $ bower install n-video --save
```

Comment:
Can we replace `-n-` with `-o-` throughout the README and it's all still applicable?